src/libZilliqa/Zilliqa.cpp

```diff
@@ -148,6 +148,41 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, SyncType syncType,
     LOG_STATE("[IDENT] " << string(key.second).substr(0, 8));
   }
 
+  // Logging useful information
+  std::string startupInfo{"Node startup Information\n"};
+
+  // 3 mode: IP/Key/P2P Seed Key (ENABLE_SEED_TO_SEED_COMMUNICATION)
+  startupInfo += "Whitelisting mode \t= ";
+  startupInfo += multiplierSyncMode
```

Comment:
`multiplierSyncMode` if true then it's IP address-based whitelisting. I think the condition should be reversed.